aspnetcore/blazor/lifecycle.md

```diff
@@ -266,3 +266,58 @@ For more information on the `RenderMode`, see <xref:blazor/hosting-model-configu
 ## Detect when the app is prerendering
 
 [!INCLUDE[](~/includes/blazor-prerendering.md)]
+
+## Cancelable background work
+
+To implement a cancelable background work pattern in a component:
```

Comment:
It would be good to provide some examples of when a user would benefit from this component as an intro here.